packages/workflow/test/TelemetryHelpers.test.ts

```diff
@@ -763,6 +765,16 @@ describe('generateNodesGraph', () => {
 			webhookNodeNames: [],
 		});
 	});
+
+	test('should not fail on error to resolve a node parameter', () => {
+		const workflow = mock<IWorkflowBase>({ nodes: [] });
```

Comment:
```suggestion
		const workflow = mock<IWorkflowBase>({ nodes: [{ type: STICKY_NODE_TYPE }] });
```

This seems to do it.

Reply:
The import still needs to be adjusted.